src/senders/report_stream_sender.go

```diff
@@ -31,10 +31,13 @@ func NewSender() (Sender, error) {
 		return Sender{}, err
 	}
 
+	privateKeySecret := "ca-phl-private-key-" + utils.EnvironmentName() // pragma: allowlist secret
```

Comment:
Calling this variable something like `privateKeyName` or `privateKeySecretName` might help future devs understand what they're looking at more easily